program_info/org.polymc.PolyMC.Source.svg

```diff
@@ -0,0 +1,261 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
What's this file for?

Reply:
its the source file that has all of the guides etc used to create the logo. its an unoptimised svg